bot/moderation/moderation.py

```diff
@@ -385,8 +385,9 @@ async def tempmute_user(self, ctx: commands.Context, user: discord.Member, durat
                                                     f"stummgeschalten.", reason, self.ch_rules)
         await user.send(embed=embed)
 
+        details = "Endet in {0} ({1})".format(pretty_duration, run_date.strftime("%d.%m.%Y %H:%M:%S"))
         modlog_embed = _build_modlog_embed("Temporäre Stummschaltung :mute:", color=constants.EMBED_COLOR_MODLOG_MUTE,
-                                           moderator=ctx.author, user=user, reason=reason)
+                                           moderator=ctx.author, user=user, reason=reason, details=details)
```

Comment:
Rename the field "details" to something like "duration" and add a point at the end of its sentence.

Reply:
But purge needs the details too. 
There it's the amount of messages and the channel.
Wanted to keep it generic if more mod commands need modlog embeds